### PR TITLE
Fix: Oos activation frequency with more multipliers to cast speed

### LINKED ISSUE #1936: Orb of Storms' hit rate is not affected by multiplicative (more/less) cast speed modifiers
In game, as per the tooltip, orb of storms' hit rate is affected by multiplicative modifiers to cast speed (i.e. "more cast speed") - this is also in accordance with the different wording of the skill gem compared to e.g. brands, which explicitly state only "increases/reductions" while OoS specifies "modifiers" to cast speed.

To confirm this, I have just tested it using essence of insanity crafted gloves. Socketing OoS into those shows a higher tooltip hit rate in accordance with the "more cast speed" modifier on the crafted gloves. In PoB, this modifier alters only OoS "cast rate", not the "hit rate".

### PR DESCRIPTION
**Pull request by pundm**
This PR tries/seems to fix #1936

image

Pastebin for testing purposes: https://pastebin.com/UkUK74xu

### CODE REVIEW

**Comment by pundm** [comment]
I just tried to create a fake jewel with less/reduced cast speed and I don't know if my solution still holds up. Could someone more knowledgeable have a look?

**Review by Wires77 — APPROVED** [review]
This looks like it's working to me

File: Data/Skills/act_int.lua [hunk 1]
```diff
@@ -6310,7 +6310,7 @@ skills["OrbOfStorms"] = {
 	statDescriptionScope = "beam_skill_stat_descriptions",
 	castTime = 0.5,
 	preDamageFunc = function(activeSkill, output)
-		activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / (1 + activeSkill.skillModList:Sum("INC", activeSkill.skillCfg, "Speed") / 100)
+		activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / (1 + activeSkill.skillModList:Sum("INC", activeSkill.skillCfg, "Speed") / 100) / activeSkill.skillModList:More(activeSkill.skillCfg, "Speed")
```
Comment (Nostrademous) [inline Data/Skills/act_int.lua:6313]: This works but calcLib.mod() could be used for the entire tail end of this to make it cleaner: https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/dev/Modules/CalcTools.lua#L16-L18

`activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / calcLib.mod(activeSkill.skillModList, activeSkill.skillCfg, "Speed")`

File: Export/Skills/act_int.txt [hunk 2]
```diff
@@ -1121,7 +1121,7 @@ local skills, mod, flag, skill = ...
 #skill OrbOfStorms
 #flags spell chaining duration
 	preDamageFunc = function(activeSkill, output)
-		activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / (1 + activeSkill.skillModList:Sum("INC", activeSkill.skillCfg, "Speed") / 100)
+		activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / (1 + activeSkill.skillModList:Sum("INC", activeSkill.skillCfg, "Speed") / 100) / activeSkill.skillModList:More(activeSkill.skillCfg, "Speed")
```
Comment (Nostrademous) [inline Export/Skills/act_int.txt:1124]: same as above

**Review by Nostrademous — CHANGES_REQUESTED** [review]

**Comment by pundm** [comment]
changed it according to your feedback and it still works. thanks!

**Review by Nostrademous — APPROVED** [review]

**Comment by Nostrademous** [comment]
You are doing great work and your help is greatly appreciated. Thanks!

**Review by ppoelzl — APPROVED** [review]
LGTM.

**Pull request merged**